chia/legacy/keyring.py

```diff
@@ -23,7 +23,7 @@
     CryptFileKeyring = None
 
 
-from chia.cmds.cmds_util import prompt_yes_no
+from chia.cmds.passphrase_funcs import prompt_yes_no
```

Comment:
This seems strictly worse.  We go from something outside of cmds using the cmds utility module to something outside of cmds using something from a cmds implementation file.  I understand you want to have pinpoint focus on only eliminating cycles.  I just don't think it's good to make the remaining dependencies be even less sensible than before to achieve that goal.  I hope we can take a little bigger picture look here and resolve this in a way that reduces cycles while not making other things distinctly worse.